modules/ossm-configuring-distr-tracing-tempo.adoc

```diff
@@ -66,35 +139,45 @@ kind: Kiali
 spec:
   external_services:
     tracing:
-      query_timeout: 30
+      query_timeout: 30 #<1>
       enabled: true
       in_cluster_url: 'http://tempo-sample-query-frontend.tracing-system.svc.cluster.local:16685'
       url: '[Tempo query frontend Route url]'
-      use_grpc: true # <1>
+      use_grpc: true # <2>
 ----
-<1> If you are not using the default HTTP or gRPC port for Jaeger or Tempo, replace the `in_cluster_url:` port with your custom port.
+<1> The default `query_timeout` is 30. If you set it greater than 30, you need to update `.spec.server.write_timeout` in the Kiali CR and add the annotation `haproxy.router.openshift.io/timeout=50s` to the Kiali route. Both `.spec.server.write_timeout` and `haproxy.router.openshift.io/timeout=` must be greater than `query_timeout`.
+<2> If you are not using the default HTTP or gRPC port for Jaeger or Tempo, replace the `in_cluster_url:` port with your custom port.
 +
 [NOTE]
 ====
 Kiali 1.73 uses the Jaeger Query API, which causes a longer response time depending on Tempo resource limits. If you see a `Could not fetch spans` error message in the Kiali UI, then check your Tempo configuration or reduce the limit per query in Kiali.
 ====
 
-. Create a TempoStack instance using the Red Hat {TempoOperator} in the `tracing-system` namespace. For more information, see "Installing the distributed tracing platform (Tempo)" in the "Additional resources" section.
+. Send requests to your application.
+
+. Verify the `istiod` pod logs and the `otel-collector` pod logs.
+
+== Configuring the {TempoName} in a mTLS encrypted Service Mesh member namespace
```

Comment:
Thinking outloud...Would this be better served as its own module? It may get lost/missed/ignored as part of `ossm-configuring-distr-tracing-tempo.adoc`

Reply:
Maybe not...since the rest of the Tempo/OTEL configuration is the same. Moving this to a separate module might be more confusing.

Reply:
If you think you'll use this procedure somewhere else, you should modularize it. You have a heading already. It will appear the same way in the production docs anyway if you call the module at the same-level heading. Seamless to the user.

Reply:
The structure, to module or not module, may be revisited for OSSM 3.0.